security/pkg/nodeagent/vm/citadelagent_v2.go

```diff
@@ -59,15 +61,22 @@ const (
 	projectIDField     = "project_id"
 )
 
+// Error messages.
+const (
+	nilConfigErr      = "node Agent configuration is nil"
+	wrongPlatformErr  = "node Agent is not running on the right platform"
+	maximumRetiresErr = "node agent can't get the CSR approved from Istio CA after max number of retries (%d)"
```

Comment:
also be consistent about node agent/node Agent, lower or capitale case.

Reply:
Done